engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetViewportSizeOf.java

```diff
@@ -86,6 +84,9 @@ public void update() {
 
     @Override
     public String toString() { // TODO: used for logging purposes at the moment, investigate different methods
+        if (isTheDefaultInstance()) { // necessary since default instance is generated before setFrameBuffersManager
```

Comment:
Let's change this to:

`if (this.isTheDefaultInstance()) { // necessary since default instance is generated before setFrameBuffersManager`

as it reads like plain English.